### PR TITLE
Make plugin chainable

### PR DESCRIPTION
**Pull request by John-Lin**
**Kubernetes 1.7+ and CNI 0.6.0-rc1 are required**
Chained plugin that supports in CNI. This PR could support the bridge plugin adds the Open vSwitch interface. Thanks to @bboreham

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,105 +53,17 @@ To provide overlay network, Linen utilize Open vSwitch to create VxLAN tunneling
...
 ### Network configuration reference
 
-For **Linux Bridge plugin** options
+Linen is a chained plugin. It always comes after `bridge` plugin, so configure Linux Bridge is needed.
+
+For the **Linux Bridge plugin** options
 - `name` (string, required): the name of the network.
 - `type` (string, required): "bridge".
-- `bridge` (string, optional): name of the bridge to use/create. Defaults to "kbr0".
+- `bridge` (string, optional): name of the bridge to use/create. Defaults to "cni0".
```
Comment (jellonek) [inline README.md]: This default seems to be different...

File: examples/master.linen.conflist [hunk 2]
```diff
@@ -0,0 +1,42 @@
+{
+   "name":"linen-network",
+   "cniVersion": "0.3.1",
+   "plugins":[
+      {
+         "type":"bridge",
+         "bridge":"kbr0",
```
Comment (jellonek) [inline examples/master.linen.conflist]: ... from this setting.
Is this correct?

Comment (John-Lin) [inline examples/master.linen.conflist]: In this example `kbr0` is not the default bridge name. If we don't set bridge name here then we'll get `cni0` by default.

Comment (jellonek) [inline examples/master.linen.conflist]: Ack.

**Pull request merged**